src/screens/NationsScreen/NationsListScreen/index.js

```diff
@@ -55,8 +55,9 @@ class NationsListScreen extends Component {
           renderItem={(item) => {
             const nation = item.item;
             const nationStatus = resolveStatus(nation);
-            const shouldShowCitizens = nationStatus === null
-              || nationStatus.code === TX_JOB_STATUS_SUCCESS;
+            // const shouldShowCitizens = nationStatus === null
+            //   || nationStatus.code === TX_JOB_STATUS_SUCCESS;
+            const shouldShowCitizens = false;
```

Comment:
Is number of citizens in nation label disabled intentionally? I believe it's a part of planned UX.

Reply:
We don't have that right now since we don't know how many citizens joined a nation.